src/spikeinterface/qualitymetrics/quality_metric_calculator.py

```diff
@@ -71,8 +84,15 @@ def _set_params(self, metric_names=None, qm_params=None, peak_sign=None, seed=No
             if "peak_sign" in qm_params_[k] and peak_sign is not None:
                 qm_params_[k]["peak_sign"] = peak_sign
 
+        metric_names_for_params = metric_names
+        qm_extension = self.sorting_analyzer.get_extension("quality_metrics")
+        if qm_extension:
```

Comment:
```suggestion
        if qm_extension is not None:
```

this is more precise right? If qm extension doesn't exist none is returned so checking for None is more precise and will help us keep track of what the code is doing. Right? Otherwise I think the ifs are better :)

Reply:
Yeah, sure. Thanks!